services/src/main/java/org/keycloak/authentication/requiredactions/WebAuthnPasswordlessRegisterFactory.java

```diff
@@ -20,13 +20,14 @@
 
 import com.webauthn4j.validator.attestation.trustworthiness.certpath.CertPathTrustworthinessValidator;
 import org.keycloak.models.KeycloakSession;
+import org.keycloak.models.UserModel;
 
 /**
  * @author <a href="mailto:mposolda@redhat.com">Marek Posolda</a>
  */
 public class WebAuthnPasswordlessRegisterFactory extends WebAuthnRegisterFactory {
 
-    public static final String PROVIDER_ID = "webauthn-register-passwordless";
+    public static final String PROVIDER_ID = UserModel.RequiredAction.WEBAUTHN_PASSWORDLESS_REGISTER.name();
```

Comment:
Is it possible to keep the Provider_ID same? I am not sure if change could affect migration (For example for these, who already have this required action enabled and registered and they migrate from previous version).

I can see for example in TermsAndConditions action that PROVIDER_ID does not need to be the name of the action on UserModel